db/seeds.rb

```diff
@@ -6,16 +6,16 @@
 User.destroy_all
 
```

Comment:
I think it would be a good idea to have another puts here to give us feedback. So that if we get an error between destroying and creating users it's easier to debug :)